src/Components/TodoContainer.js

```diff
@@ -53,14 +53,28 @@ function TodoContainer() {
     // Fetch data on initial render and whenever sorting order changes
     useEffect(() => {
         fetchData();
+        
     }, [fetchData]);
 
-    // Function to toggle sorting order
+    // Function to toggle sorting order and sort the todoList accordingly
     function toggleSortingOrder() {
         // Toggle between ascending and descending order
         setIsAscending(!isAscending);
+
+        // Sort the todoList based on the new sorting order
+        setTodoList((prevTodoList) =>
+            [...prevTodoList].sort((a, b) => {
+                if (isAscending) {
+                    return a.createdTime < b.createdTime ? -1 : 1;
+                } else {
+                    return a.createdTime > b.createdTime ? -1 : 1;
+                }
```

Comment:
I do this all the time, but you forgot to update the code after you pasted it, so you've got the `if` and the `else` running the same code

Reply:
this also maybe shows that it could be helpful to do some more testing in your browser before you submit the PR ;)

Reply:
> I do this all the time, but you forgot to update the code after you pasted it, so you've got the `if` and the `else` running the same code

I'm sorry, I don't quite understand what you mean. The `if` statement sorts the items in ascending order based on `createdTime` while the `else` statement does the opposite.

Reply:
I'm sorry this is my fault misreading the code here! 

I always do the `a.createdTime - b.createdTime` variant of the sort logic, so I missed that you were correctly using `<` and `>` here!